wordle-web/components/SettingsDialog.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-dialog v-model="modelValue">
+    <v-card class="ma-auto pa-4" min-width="300">
+      <v-sheet color="secondary">
+        <v-card-title>Settings</v-card-title>
+      </v-sheet>
+      <v-card-text>
+        <v-divider class="my-4" />
+        Light/Dark Mode:
+        <v-btn
+          color="secondary"
+          @click="toggleTheme"
+          icon="mdi-theme-light-dark" />
+        <v-divider class="my-4" />
+      </v-card-text>
+
+      <v-sheet class="ml-6" width="200">
+        <v-form @submit.prevent>
+          <v-select
+            color="secondary"
+            v-model="selectedTheme"
+            label="Select theme"
+            :items="['Regular', 'Jungle', 'Flamingo', 'Whurdle']"></v-select>
+          <v-btn
+            color="secondary"
+            @click="changeColorScheme"

```

Comment:
Could have this also toggle ShowSettingsDialog from default.vue

Reply:
I think submitting the color theme shouldn't close the settings dialog, as the user might want to change the dark/light mode. Thank you for the suggestion though :)